<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_window</class>
 <widget class="QDialog" name="search_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Task name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="task_name_area"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Task details:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="task_details_area"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>161</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>priority :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="priority_area"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>351</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="day_area"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="month_area"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="year_area"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>tasknumber:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="task_number_area"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
